cfn-resources/custom-dns-configuration-cluster-aws/README.md

```diff
@@ -0,0 +1,64 @@
+# MongoDB::Atlas::CustomDnsConfigurationClusterAws
+
+## Description
+Returns, adds and removes custom dns configuration for AWS cluster.
+
+## Attributes & Parameters
+
+Please consult the [Resource Docs](https://github.com/PeerIslands/mongodbatlas-cloudformation-resources/blob/feature-custom-dns-config-cluster-aws/cfn-resources/custom-dns-configuration-cluster-aws/docs/README.md)
+
+## Local Testing
+
+The local tests are integrated with the AWS sam local and cfn invoke tooling features:
+
+```
+sam local start-lambda --skip-pull-image
+```
+then in another shell:
+```bash
+#https://www.mongodb.com/docs/mongocli/stable/configure/environment-variables/
+#Set the public API key for commands that interact with your MongoDB service.
+export MCLI_PUBLIC_API_KEY = ""
+#Set the private API key for commands that interact with your MongoDB service.
+export MCLI_PRIVATE_API_KEY=""
+#Sets the project ID for commands that require the --projectId option.
+export MCLI_PROJECT_ID = ""
+
+cd ${repo_root}/cfn-resources/custom-dns-configuration-cluster-aws
+./test/custom-dns-config-cluster-aws.create-sample-cfn-request.sh > test.request.json
+echo "Sample request:"
+cat test.request.json
+cfn invoke CREATE test.request.json
+cfn invoke DELETE test.request.json
+```
+
+Both CREATE & DELETE tests must pass.
+
```

Comment:
Missing Example Section, can you add and link to Example directory: https://github.com/mongodb/mongodbatlas-cloudformation-resources/tree/master/examples